docs/design/marketing-offer-program.md

```diff
@@ -29,12 +29,12 @@ Existing clients give their UUID Referral Link to their friends for their friend
 ### AWARDED CREDIT
 
 - When free credit is awarded:
-It is displayed as the USD value that is equivalent to the storage & bandwidth allotted amount for the credit
+It is displayed as the Network activity value (gb) that is equivalent to the storage & bandwidth allotted amount for the credit
 The credit is automatically applied to the account and will have a max limit that can be reached within a set timeframe, as well as an expiration date for it to be used by.
   The user will be charged for activity that is beyond the Free Credit limit within the set timeframe
   The user will be charged for activity that is after the expiration date.
   Example statement to the user: 
-  “You will not be charged until all of your $XX credit is used or it expires. Your $XX credit will be applied immediately”
+  “You will not be charged until all of your XX credit is used or it expires. Your $XX credit will be applied immediately”
```

Comment:
Forgot to remove second dollar sign?

Reply:
Business just confirmed that `credits` will be using `USD` as its unit. I'm reverting it back

Reply:
Removed it

Reply:
nit-pick: The sentence reads fine w/o the `$xx` but I know it's just an example statement